packages/admin-ui/src/Radio/RadioGroupPresenter.ts

```diff
@@ -0,0 +1,48 @@
+import { makeAutoObservable } from "mobx";
+import { RadioGroupVm } from "./RadioGroup";
+import { RadioItem } from "~/Radio/RadioItem";
+import { RadioItemMapper } from "~/Radio/RadioItemMapper";
+import { RadioItemDto } from "~/Radio/RadioItemDto";
+
+interface RadioGroupPresenterParams<TValue = string> {
+    items: RadioItemDto[];
+    onValueChange: (value: TValue) => void;
+}
+
+interface IRadioGroupPresenter<TValue = string> {
+    vm: RadioGroupVm;
+    init: (props: RadioGroupPresenterParams<TValue>) => void;
+    changeValue: (value: TValue) => void;
+}
+
+class RadioGroupPresenter<TValue = string> implements IRadioGroupPresenter<TValue> {
+    private params?: RadioGroupPresenterParams<TValue> = undefined;
+
+    constructor() {
+        makeAutoObservable(this);
+    }
+
+    public init = (params: RadioGroupPresenterParams<TValue>) => {
+        this.params = params;
+    };
+
+    get vm() {
+        return {
+            items: this.items.map(item => RadioItemMapper.toFormatted(item))
+        };
+    }
+
+    public changeValue = (value: TValue) => {
+        this.params?.onValueChange(value);
+    };
+
+    private get items() {
```

Comment:
I'd kinda want to see a more explicit name here because this is not just a simple getter. Maybe something like `mapItemsToRadioItems` or maybe `getRadioItemsFromParams`. 

Jus something that crossed my mind. But yeah, more of a nitpick, especially since this is a private method.

Reply:
It makes sense, I changed it to `getRadioItems`